### PR TITLE
Support URI data scheme in Image

### PR DESCRIPTION
**Pull request by trieutritam**

### CODE REVIEW

File: lib/src/builder.dart [hunk 1]
```diff
@@ -260,7 +260,11 @@ class MarkdownBuilder implements md.NodeVisitor {
     Widget child;
     if (uri.scheme == 'http' || uri.scheme == 'https') {
       child = new Image.network(uri.toString(), width: width, height: height);
-    } else {
+    }
+    else if (uri.scheme == 'data') {
```
Comment (Hixie) [inline lib/src/builder.dart]: nit: here and elsewhere -- please put the `else` on the same line as the `}`, as in `} else if (...) {` or `} else {`

File: lib/src/builder.dart [hunk 2]
```diff
@@ -275,6 +279,26 @@ class MarkdownBuilder implements md.NodeVisitor {
...
+      child = new Image.memory(uri.data.contentAsBytes(), width: width, height: height);
+    }
+    else if (mimeType.startsWith('text/')) {
+      child = new Text(uri.data.contentAsString());
+    }
+    else {
+      child = const SizedBox();
+    }
+
+
```
Comment (Hixie) [inline lib/src/builder.dart]: nit: extraneous blank line here.

Actually you could remove all the blank lines in this function, they don't really add much. If you wanted you could even further and remove the `child` variable, and just `return` straight from each `if` block (and then drop the `else` keywords also).

**Comment by Hixie** [comment]
This looks great, and especially thanks for the tests! Just a few nits. Let me know when you've addressed them and I can land the patch! Thanks!

**Comment by Hixie** [comment]
LGTM

**Comment by trieutritam** [comment]
@Hixie Thank you. I've changed code as your comments.

**Comment by Hixie** [comment]
Thanks for your contribution!

**Pull request merged**